README.md

```diff
@@ -33,7 +33,7 @@ They are controled from the tray, no additional configuration is necessary.
 
 #### Autostart
 
-Using `~/.config/autostart` is preferable, which is used by "Autostart" in KDE settings or Gnome Tweaks.
+When using the Bundle using `~/.config/autostart` is preferable, which is used by "Autostart" in KDE settings or Gnome Tweaks.
```

Comment:
But this is a subsection of "_Bundle with built-in ActivityWatch_" which already suggests the bundle.
```suggestion
It is recommended to utilize `~/.config/autostart` for the bundle. This folder is employed by "Autostart" in KDE settings and Gnome Tweaks.
```